docs/maintain-guides-how-to-build-validator-community.md

```diff
@@ -0,0 +1,20 @@
+---
+id: maintain-guides-how-to-build-validator-community
+title: Build a Community for Your Validator
+sidebar_label: Build a Community for Your Validator
+description: Build a Community for Your Validator
+---
+
+Validators are welcome to promote your node in the channels listed below or other social media
+platforms to let more people know about what is about your validator, what have you done for the
+network, why nominators should support your work. There are multiple ways to help you to get
+nominations from the community, for example, write a staking tutorial in a specific language that
+targets those communities, provide staking tooling to the community, etc..
```

Comment:
I would reword this part as a list. So for example:
`There are multiple ways to obtain nominations from the community, some of them being:
- list 1 
- list 2
`

Reply:
I'd also recommend here that another idea is to answer peoples questions they have on social media accounts. The more people recognize your name, the more they might be willing to nominate for you.